### ISSUE TITLE
can we enter multiple countries in country code parameter

### ISSUE THREAD

**Issue by Samkit02** [larryaasen/upgrader#129]
hello,

great work creating this package

my question is - can we enter multiple countries in country code parameter ??

**Comment by larryaasen** [comment]
I assume the array of countries would be used to query the App Store in array order, and when the first store returns a newer version, that would be used as the update version. However, when a user upgrades an app using the App Store, the country used by the App Store is based on the Country/Region setting in the account App Store account. If the wrong country code is used the app will not update. 
If you have released the same version of your app to all of those countries, then the user will be prompted for their own country. Why does this not work in your case?

**Comment by Samkit02** [comment]
> Did you try it?

yes i have entered the code but i have havent uploaded new version to playstore so after that i can check if it works or not

**Comment by larryaasen** [comment]
@Samkit02 Do you still have this issue?

**Comment by Samkit02** [comment]
> @Samkit02 Do you still have this issue?

havent tested yet
